### ISSUE TITLE
Failure due to "Unknown log line" exception

### ISSUE THREAD

**Issue by mbluj** [kandrosov/RunKit#15]
My `crabOverseer.py` execution fails due to an "Unknown log line" exception thrown by `crabTaskStatus.py`. The exception was caused by unexpected line "Rucio client intialized for account [account name]" in output of `crab status` command.
It is not clear for me if the line is present because of some changes (updates) in Rucio service or because of my username on a local machine is different than my grid username. Regardless on the reason, the issue can be common as my setup is not special (`NanoProd` tools run at lxplus8). 
Therefore, I would like to propose the following fix of the issue. I do not want to clone this repo and prepare PR just for this small fix and it is why I go trough issue.
```
diff --git a/crabTaskStatus.py b/crabTaskStatus.py
index b11d4af..4d54c3d 100644
--- a/crabTaskStatus.py
+++ b/crabTaskStatus.py
@@ -325,6 +325,7 @@ class LogEntryParser:
     "Warning:": warning,
     "Jobs status:": job_status,
     "No publication information": None,
+    "Rucio client intialized for account": None,
     "Error Summary:": error_summary,
     "Log file is": "crab_log_file",
     "Summary of run jobs:": run_summary,
```

**Comment by kandrosov** [comment]
Hi @mbluj . Isn't it already fixed in the PR https://github.com/kandrosov/RunKit/pull/14?

**Comment by mbluj** [comment]
> Hi @mbluj . Isn't it already fixed in the PR #14?

Indeed. I have version of the tools before this PR. I will update to the master.
Closing this issue.
